src/simple/room/Room.js

```diff
@@ -346,6 +350,25 @@ class Room extends Emitter {
     Wazo.Phone.sendDTMF(tone, this.callSession);
   }
 
+  async sendReinvite(sipSession: Session, newConstraints: Object = null) {
+    logger.info('send room reinvite', { sipSession, newConstraints });
+
+    await Wazo.Phone.phone.sendReinvite(sipSession, newConstraints, true);
+
+    if (this.localParticipant && newConstraints && newConstraints.video) {
+      const pc = sipSession.sessionDescriptionHandler.peerConnection;
+      pc.getSenders()
+        .filter(localSender => localSender.track && localSender.track.kind === 'video')
+        .forEach(localVideoSender => {
+          const mediaStream = new MediaStream();
```

Comment:
I'm not sure it should be a new `MediaStream`: it should be synchronized with audio sender, so IMHO it should be the same `MediaStream` than audio, but I couldn't find how to get the same `MediaStream`

Reply:
We'll take a look when dealing with reinvite in SFU mode.